tools/generators/epic-generator/index.ts

```diff
@@ -0,0 +1,191 @@
+import { getProjects, stripIndents, Tree } from '@nrwl/devkit';
+import { readFileSync } from 'fs';
+import { execSync } from 'child_process';
+import { EpicGeneratorSchema } from './schema';
+import { isPackageConverged } from '../../utils';
+
+function validateSchema(schema: EpicGeneratorSchema): Required<EpicGeneratorSchema> {
+  if (schema.repository !== undefined && !schema.repository.match(/[A-z-]+\/[A-z-]+/)) {
+    throw new Error('Must provide a valid repository with the format {owner}/{repositoryName}');
+  }
+
+  if ((schema.title || '').trim() === '') {
+    throw new Error('Must provide a title for the issue');
+  }
+
+  return {
+    repository: schema.repository || 'microsoft/fluentui',
+    title: schema.title,
+    message: schema.message || '*Description to be added*',
+  };
+}
+
+const checkAuthentication = () => {
+  let authStatus;
+
+  try {
+    authStatus = execSync('ghz auth status', { stdio: [] }).toString();
```

Comment:
is killing default process output on purpose here ? `{ stdio: []}`

Reply:
Definite oversight. The idea was to avoid piping stdout and stderr to avoid unnecessary terminal clutter, but I did not account for the execSync function to not return it. Gotta research how to do that 🕵️

Reply:
I got to the root of the problem. `gh auth status` sends multiple lines of output and `execSync` only parses the first, resulting in an empty response always. The reason this was passing was because we were checking for the message to include "You are not logged into any GitHub hosts", which meant that any other message would not throw an error.
I refactored this to use `spawnSync` and parse the whole output, as well as check if the message does NOT contain "Logged in to github.com", which is the only case where we want to pass.